finality-aleph/src/sync/forest/mod.rs

```diff
@@ -146,6 +160,8 @@ where
     vertices: HashMap<BlockIdFor<J>, VertexWithChildren<I, J>>,
     highest_justified: BlockIdFor<J>,
     justified_blocks: HashMap<u32, BlockIdFor<J>>,
+    imported_leaves: HashSet<BlockIdFor<J>>,
```

Comment:
not to this line but to the #162. why do we have map with u32 :fearful:

Reply:
Hm, that is equal to `BlockNumber` anyway, but yeah, changed to that explicitly. Also changed in a couple other places.